lib/widgets/device_selector_screen.dart

```diff
@@ -27,13 +27,43 @@ import 'package:flutter/material.dart';
 import 'package:flutter_gen/gen_l10n/app_localizations.dart';
 import 'package:provider/provider.dart';
 
+Future<Device?> showDeviceSelectorScreen(
```

Comment:
This needs to have some rework to match the overall app design and behavior. It works as it is tho